test/unit/helper.spec.ts

```diff
@@ -1,15 +1,24 @@
 import assert from "assert";
-import { isResourceValid } from "../../dist-esm/common";
+import { isResourceValid, parseConnectionString } from "../../dist-esm/common";
 
 describe("Helper methods", function() {
   describe("isResourceValid Unit Tests", function() {
-    it("id is not string", function(done) {
+    it("id is not string", function() {
       const err = {};
       const result = isResourceValid({ id: 1 }, err);
 
       assert.equal(result, false);
       assert.deepEqual(err, { message: "Id must be a string." });
-      done();
+    });
+  });
+  describe("parseConnectionString", function() {
+    it("parses a valid connection string", function() {
```

Comment:
Please add case for how the error message looks/etc. when it is an invalid connection string.